variables.tf

```diff
@@ -2,8 +2,42 @@
 # Input Variables
 ########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "watsonx_data_name" {
+  type        = string
+  description = "The name of the watsonx.data instance."
+}
+
+variable "resource_group_id" {
+  description = "The resource group ID where the Event Streams instance is created."
+  type        = string
+}
+
+variable "region" {
+  default     = "us-south"
+  description = "The region that's used with the IBM Cloud Terraform IBM provider. It's also used during resource creation."
```

Comment:
The descrption should include all supported values. Are we sure this list is valid?